src/Devolutions.AvaloniaTheme.DevExpress/Devolutions.AvaloniaTheme.DevExpress.csproj

```
@@ -12,8 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="11.2.7" />
-    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.2.7" />
-    <PackageReference Include="Avalonia.Controls.DataGrid" Version="11.2.7" />
+    <PackageReference Include="Avalonia" Version="[11.2.0,)" />
+    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.2.8" />
```

If the goal is to pin to this specific tested version as it is an internal dependency, then is should be this instead:
```suggestion
    <PackageReference Include="Avalonia.Themes.Fluent" Version="[11.2.8]" />
```

(see: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges)

---

duh! I read it and straight away forgot again! (to my mind I always thought the plain number would nail it down)
Thanks for paying attention! 😆

---

Fixed